### PR TITLE
Feat: Nuvem shop integration

### PR DESCRIPTION
**Pull request by rafaeltj13**
This PR intents to create and integrate 3 loader functions with nuvemShop API:

- nuvemShopProductListingPage;
- nuvemShopProductList;
- nuvemShopProductDetailsPage.

I've faced some limitations regarding the the nuvemShop API so some functionalities like sort and filter in nuvemShopProductListingPage are commented.

### CODE REVIEW

File: sections/configNuvemShop.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import type { ConfigNuvemShop } from "../commerce/nuvemShop/types.ts";
+
+function ConfigSection(_: ConfigNuvemShop) {
```
Comment (lucis) [inline sections/configNuvemShop.tsx:3]: Use Accounts instead

File: .frsh/js/metafile.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
+{
```
Comment (lucis) [inline .frsh/js/metafile.json:1]: Remove `.frsh/*`

File: commerce/nuvemShop/client.ts [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  if (!params) return;
+
+  const fetcher = <T>(
+    endpoint: string,
+    method = "GET",
+    data?: Record<string, unknown>,
+  ) => {
+    const { userAgent, accessToken, storeId } = params;
+
+    console.log(`${BASE_URL}/${storeId}/${endpoint}`);
```
Comment (lucis) [inline commerce/nuvemShop/client.ts:54]: Remove

File: loaders/nuvemShop/nuvemShopProductDetailsPage.ts [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+  if (!nuvemProduct) {
+    return null;
+  }
+
+  const [product] = toProduct(nuvemProduct, new URL(req.url), sku);
+
+  return {
+    "@type": "ProductDetailsPage",
+    // TODO: Find out what's the right breadcrumb on nuvem shop
```
Comment (lucis) [inline loaders/nuvemShop/nuvemShopProductDetailsPage.ts:36]: Did you?

**Review by lucis — APPROVED** [review]

File: loaders/nuvemShop/nuvemShopProductDetailsPage.ts (hunk 4 above)
Comment (rafaeltj13) [reply loaders/nuvemShop/nuvemShopProductDetailsPage.ts:36]: Yes, this is a old comment

**Pull request merged**